helm/tenant/templates/tenant.yaml

```diff
@@ -136,4 +136,43 @@ spec:
   {{ if dig "priorityClassName" "" . }}
   priorityClassName: {{ dig "priorityClassName" "" . }}
   {{ end }}
+  {{ if dig "kes" "enabled" false . }}
```

Comment:
Can we check here the `kesSecret.name` field is defined instead of adding an additional "enabled" field to kes?

Reply:
Sure thing, working into that